cephfs/admin/fsadmin.go

```diff
@@ -143,3 +143,18 @@ func modeString(m int, force bool) string {
 func uint64String(v uint64) string {
 	return strconv.FormatUint(uint64(v), 10)
 }
+
+type rmFlags struct {
+	force bool
+}
+
+func (f rmFlags) Update(m map[string]string) map[string]interface{} {
+	o := make(map[string]interface{})
```

Comment:
Is it really needed to make a copy of the map?

Reply:
Yeah. Unfortunately go can't "typecast" between map[string]string -> map[string]interface{} even though interface{} is kind-of the anything type.